app/views/users/_edit_form.html.erb

```diff
@@ -33,18 +33,25 @@
 
       <div class="form-group col-md-6">
         <label for="email"><%= translation('users._form.email') %></label>
-        <%= f.text_field :email, { tabindex: 3, required: true, placeholder: "you@email.com", class: 'form-control', id: 'email'} %>
+        <%= f.text_field :email, { tabindex: 3, required: true, placeholder: "you@email.com", class: 'form-control', id: 'new_email'} %>
```

Comment:
Hi @sakshi-2412, amazing work! I just wanted to know why you changed the email field's id.